docs/config/README.md

```diff
@@ -4684,6 +4684,128 @@ By default the module will be shown if any of the following conditions are met:
 format = 'via [V $version](blue bold) '
 ```
 
+## VCS
+
+The `vcs` module displays the current active Version Control System (VCS).
+The module will be shown only if a VCS is currently in use.
+
+### Options
+
+| Option      | Default       | Description                         |
+| ----------- | ------------- | ----------------------------------- |
+| `order`     | `["jujutsu"]` | The order in which to search VCSes. |
+| `jujutsu`   | _see below_   | The Jujutsu configuration.          |
+| `mercurial` | _see below_   | The Mercurial configuration.        |
+| `disabled`  | `true`        | Disables the `vcs` module.          |
+
+Note that an empty `order` has the same effect as disabling the module.
+
+VCS will be searched for following `order`. The first one that returns valid informations will be the one printed,
+e.g. in a repository with both `.git` and `.jj` present, `order = ["jujutsu", "git"]` will always print Jujutsu's output.
+
```

Comment:
As a user, I'd be confused as to why there's no `git` listed in the Options table. If git can be configured here, that would be ideal, but if not, the README should tell you where to go instead.

Also, I thought the plan was not to make `["jujutsu"]` the default ordering. Should that not be `["git"]`, instead?